server/utils/utils.ts

```diff
@@ -94,3 +105,16 @@ export const dateAndTimeInputsAreValidDates = <K extends string | number>(
 
   return true
 }
+/**
+ * Retrieves response for a given question from the session object.
+ * @param sessionData the session data for an application.
+ * @param question the question that we need the response for in camelCase.
+ * @returns name converted to proper case.
+ */
+export const retrieveQuestionResponseFromSession = <T>(sessionData: Record<string, unknown>, question: string) => {
```

Comment:
What happens if the question name is different from the page name?

Reply:
Currently we can't handle that scenario but we could add an optional third parameter if it's needed

Reply:
It's probably not worth it TBH, as we're going to change the structure